src/test/java/apicurito/tests/utils/slenide/CommonUtils.java

```diff
@@ -81,11 +90,56 @@ public static void setValueInLabel(String value, SelenideElement section, boolea
     }
 
     public static void setDropDownValue(String buttonId, String value, SelenideElement section) {
-        log.info("Setting value {} in dropdown {} in section{}", value, buttonId, section.getValue());
+        log.info("Setting value {} in dropdown {} in section {}", value, buttonId, section.getAttribute("class"));
         section.$(buttonId).click();
         section.$$("li").filter(text(value)).first().click();
     }
 
+    /**
+     * It works for creating: Query parameters (both), Header parameters (both), Reqest form data, Data type property
+     */
+    public static void fillEntityEditorForm(DataTable table) {
+        SelenideElement newDataFormPage = MainPageUtils.getMainPageRoot().$("#entity-editor-form");
+
+        for (List<String> dataRow : table.cells()) {
+            newDataFormPage.$("#qp_name").setValue(dataRow.get(0));
+
+            if (!dataRow.get(1).isEmpty()) {
+                newDataFormPage.$("ace-editor textarea").sendKeys(dataRow.get(1));
+            }
+
+            if (!dataRow.get(2).isEmpty()) {
+                CommonUtils.setDropDownValue(CommonUtils.DropdownButtons.PARAM_REQUIRED.getButtonId(), Boolean.valueOf(dataRow.get(2)) ? "Required" : "Not Required", newDataFormPage);
+            }
+
+            if (!dataRow.get(3).isEmpty()) {
+                CommonUtils.setDropDownValue(CommonUtils.DropdownButtons.PARAM_TYPE.getButtonId(), dataRow.get(3), newDataFormPage);
```

Comment:
If you send value of `DropdownButtons` from data table to this method you can use a cycle to fill all those dropdowns - would be much nicer code.